tests/API_Amazon.py

```diff
@@ -0,0 +1,48 @@
+# Import and load necessary lib
+from bs4 import BeautifulSoup
+import requests
+
+
+def extract_jobs_url(soup):
+    fb_url = "https://www.facebook.com"
```

Comment:
still a copy of FB_API, not converted to amazon yet

Reply:
Last time we talked about the API_RH - it's working now, Amazon still in progress